exchanges/zb/zb_websocket.go

```diff
@@ -144,11 +78,12 @@ func (z *ZB) WsHandleData() {
 	for {
 		select {
 		case <-z.Websocket.ShutdownC:
-
+			return
 		default:
 			resp, err := z.WsReadData()
 			if err != nil {
 				z.Websocket.DataHandler <- err
+				time.Sleep(time.Second)
```

Comment:
Why do we sleep here?

Reply:
When there is no connection and you try to read data, you get an immediate error. If I don't sleep here, then the applications messages so many errors the application crashes

Reply:
makes sense